src/libraries/System.Private.CoreLib/src/System/SpanHelpers.cs

```diff
@@ -31,6 +31,7 @@ public static void ClearWithoutReferences(ref byte b, nuint byteLength)
                 {
                     do
                     {
+                        Vector<byte>.Zero.StoreUnsafe(ref b, offset);
```

Comment:
why do we store twice to the same location?

Reply:
bad git conflict resolve..